specification/v1.4.0-rc/OSDM-offline-model-v1.4.0-rc.json

```diff
@@ -1,2246 +1,2362 @@
+					"type": "object",
+					"properties": {
+						"latestReturn": {
+							"description": "number of days after departure or start of validity of the last return",
+							"type": "integer",
+							"format": "int32"
+						},
+						"earliestReturn": {
+							"description": "number of days after departure or start of validity of the earliest return",
+							"type": "integer",
+							"format": "int32"
+						},
+						"excludedWeekdays": {
+							"description": "weekdays (ISO day of the week, 1 = Monday) between travel and return where travel is not allowed",
+							"type": "array",
+							"items": {
+								"type": "integer",
+								"format": "int32"
+							}
+						}
+					},
+					"required": [
+						"latestReturn",
+						"earliestReturn"
+					],
+					"additionalProperties": false
+				},
+				"trainValidity": {
+					"description": "validity depending on boarding / leaving of a train. The ticket is also valid from departure until the destination station in the train ",
+					"type": "object",
+					"properties": {
+						"carrierConstraintRef": {
+							"type": "string"
+						},
+						"serviceConstraintRef": {
+							"type": "string"
+						},
+						"scope": {
+							"type": "string",
+							"enum": [
+								"BOARDING",
+								"ARRIVAL"
+							],
+							"default": "BOARDING"
+						}
+					},
+					"required": [
+						"carrierConstraintRef",
+						"scope"
+					],
+					"additionalProperties": false
+				},
+				"validityType": {
+					"description": "type of usage, eigther as single trip or multiple trips. Unrestricted fares might be combined into single trips when appropriate",
+					"type": "string",
+					"default": "SINGLE_TRIP",
+					"x-extensible-enum": [
+						"SINGLE_TRIP",
+						"MULTIPLE_TRIPS",
+						"UNRESTRICTED"
+					]
+				},
+				"tripAllocationConstraint": {
+					"$ref": "#/definitions/TripAllocationConstraintDef"
+				},
+				"tripInterruptionConstraint": {
+					"$ref": "#/definitions/TripInterruptionConstraintDef"
+				}
+			},
+			"required": [
+				"validityRange"
+			],
+			"additionalProperties": false
+		},
+		"TripInterruptionProcess": {
+			"description": "MANUAL: passenger needs manual confirmation of train or station staff to interrupt the trip. ACTIVATION: Passenger needs to deactivate the ticket during the interruption (via an app).",
+			"type": "string",
+			"x-extensible-enum": [
+				"NONE",
+				"MANUAL",
+				"ACTIVATION"
+			]
+		},
+		"TripAllocationProcess": {
+			"description": "NONE: Individual trips don't need to be allocated. MANUAL: The passenger has to allocate a trip manually (This should not be used as it is by desiugn non-interoperable). ACTIVATION: Individual trips can be allocatted by activation of the ticket. FULFILLMENT A new fulfillment needs to be created to allocate a trip",
+			"type": "string",
+			"x-extensible-enum": [
+				"NONE",
+				"MANUAL",
+				"ACTIVATION",
+				"FULFILLMENT"
```

Comment:
Do we have a position on US English vs UK English? The version with to `l` is american English. https://en.wiktionary.org/wiki/fulfilment

Reply:
I'm using the CodeSpellChecker plugin in VSCode set to US-English

Reply:
Good questions. en_GB would at least ensure I don't need to double check each time if double L is written after first or second F. :) And I was also corrected once incorrectly that the correct spelling is fullfiled. :/